Tests/YPersistenceTests/SortColumnTests.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SortColumnTests.swift
+//  YPersistence
+//
+//  Created by Sahil Saini on 13/04/23.
+//  Copyright © 2023 Y Media Labs. All rights reserved.
+//
+
+import XCTest
+@testable import YPersistence
+
+final class SortColumnTests: XCTestCase {
+    func test_defaultAscendingOrder_isTrue() {
+        let sut = makeSut()
+        XCTAssertTrue(sut.ascending)
+    }
+
+    func test_descriptorKeyOrder_isCorrect() {
+        let expectedKey = "Mango"
+        let sut = makeSut(key: expectedKey, ascending: false)
+
+        XCTAssertEqual(sut.descriptor.key, expectedKey)
+        XCTAssertFalse(sut.descriptor.ascending)
+    }
```

Comment:
```suggestion
    func test_descriptor_deliversCorrectValues() {
        let key = "Mango"
        let ascending = Bool.random()
        let sut = makeSut(key: key, ascending: ascending)
        let expected = NSSortDescriptor(key: key, ascending: ascending)

        XCTAssertEqual(sut.descriptor, expected)
    }
```
Does NSSortDescriptor conform to Equatable? If so, I'd prefer the above.
Even if not would be good to use Bool.random to assign `ascending`